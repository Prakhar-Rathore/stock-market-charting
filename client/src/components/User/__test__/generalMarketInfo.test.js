import React from "react";
import ReactDOM from "react-dom/client";
import GeneralMarketInfo from "./../generalMarketInfo";
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import { act } from 'react-dom/test-utils';

it("renders without crashing", async() => {
  const root = ReactDOM.createRoot(document.createElement("div"));
  await act( async () => render(<GeneralMarketInfo/>));
});

it("renders correctly", () => {
  render(<GeneralMarketInfo/>)
})

it("matches snapshot", () => {
  const tree = renderer.create(<GeneralMarketInfo/>).toJSON();
  expect(tree).toMatchSnapshot();
})
