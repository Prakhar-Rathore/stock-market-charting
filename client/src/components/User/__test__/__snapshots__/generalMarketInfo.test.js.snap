// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
Array [
  <div
    className="MuiBox-root css-1cako5s"
  >
    <div
      className="MuiGrid-root MuiGrid-item css-1jgz0p5-MuiGrid-root"
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
        data-testid="SearchIcon"
        focusable="false"
        style={
          Object {
            "coSelf": "center",
            "margin": "5px",
          }
        }
        viewBox="0 0 24 24"
      >
        <path
          d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
        />
      </svg>
      <div
        aria-owns={null}
        className="MuiAutocomplete-root MuiAutocomplete-hasPopupIcon css-16awh2u-MuiAutocomplete-root"
        color="primary"
        onClick={[Function]}
        onKeyDown={[Function]}
        onMouseDown={[Function]}
        style={
          Object {
            "border": "1px solid white",
            "borderRadius": "15px",
            "padding": "10px",
            "width": "75%",
          }
        }
      >
        <div
          className="MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-adornedEnd MuiAutocomplete-inputRoot css-hs4fzq-MuiInputBase-root"
          onClick={[Function]}
        >
          <input
            aria-activedescendant={null}
            aria-autocomplete="list"
            aria-expanded={false}
            autoCapitalize="none"
            autoComplete="off"
            className="MuiInputBase-input MuiInputBase-inputAdornedEnd MuiAutocomplete-input MuiAutocomplete-inputFocused css-156xqnd-MuiInputBase-input"
            disabled={false}
            id="input"
            onAnimationStart={[Function]}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            onMouseDown={[Function]}
            placeholder="Search for a company..."
            readOnly={false}
            role="combobox"
            spellCheck="false"
            type="text"
            value=""
          />
          <div
            className="MuiAutocomplete-endAdornment css-1q60rmi-MuiAutocomplete-endAdornment"
          >
            <button
              aria-label="Open"
              className="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium MuiAutocomplete-popupIndicator css-1ciwbrp-MuiButtonBase-root-MuiIconButton-root-MuiAutocomplete-popupIndicator"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onContextMenu={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={-1}
              title="Open"
              type="button"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="ArrowDropDownIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M7 10l5 5 5-5z"
                />
              </svg>
              <span
                className="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
      </div>
    </div>
    <button
      aria-label="toggle between light and dark mode"
      aria-labelledby={null}
      className="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root  css-gjqkap-MuiButtonBase-root-MuiButton-root"
      data-mui-internal-clone-element={true}
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onContextMenu={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseOver={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
        data-testid="DarkModeIcon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M12 3c-4.97 0-9 4.03-9 9s4.03 9 9 9 9-4.03 9-9c0-.46-.04-.92-.1-1.36-.98 1.37-2.58 2.26-4.4 2.26-2.98 0-5.4-2.42-5.4-5.4 0-1.81.89-3.42 2.26-4.4-.44-.06-.9-.1-1.36-.1z"
        />
      </svg>
      <span
        className="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
      />
    </button>
  </div>,
  <div
    className="MuiGrid-root css-1up9x2w-MuiGrid-root"
  >
    <span
      className="MuiTypography-root MuiTypography-body3 css-n2jbul-MuiTypography-root"
    >
      Market Summary
    </span>
  </div>,
  <div
    className="MuiBox-root css-fo6wgm"
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-spacing-md-2 MuiGrid-spacing-lg-2 css-1ffx5kp-MuiGrid-root"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 MuiGrid-grid-sm-4 MuiGrid-grid-md-4 MuiGrid-grid-lg-5 css-b8gvhe-MuiGrid-root"
      >
        <div
          className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-1xsv8fd-MuiPaper-root"
        >
          <div
            className="MuiGrid-root css-12ja97j-MuiGrid-root"
          >
            <p
              className="MuiTypography-root MuiTypography-body2 css-1g1qlhj-MuiTypography-root"
            >
              NIFTY 50
            </p>
          </div>
          <div />
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 MuiGrid-grid-sm-4 MuiGrid-grid-md-4 MuiGrid-grid-lg-5 css-b8gvhe-MuiGrid-root"
      >
        <div
          className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-1xsv8fd-MuiPaper-root"
        >
          <div
            className="MuiGrid-root css-vj1n65-MuiGrid-root"
          >
            <p
              className="MuiTypography-root MuiTypography-body2 css-1g1qlhj-MuiTypography-root"
            >
              BSE SENSEX
            </p>
          </div>
          <div />
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 MuiGrid-grid-sm-4 MuiGrid-grid-md-4 MuiGrid-grid-lg-5 css-b8gvhe-MuiGrid-root"
      >
        <div
          className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-1xsv8fd-MuiPaper-root"
        >
          <div
            className="MuiGrid-root css-vj1n65-MuiGrid-root"
          >
            <p
              className="MuiTypography-root MuiTypography-body2 css-1g1qlhj-MuiTypography-root"
            >
              DIRECTION
            </p>
          </div>
          <div />
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 MuiGrid-grid-sm-4 MuiGrid-grid-md-4 MuiGrid-grid-lg-5 css-b8gvhe-MuiGrid-root"
      >
        <div
          className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-1xsv8fd-MuiPaper-root"
        >
          <div
            className="MuiGrid-root css-vj1n65-MuiGrid-root"
          >
            <p
              className="MuiTypography-root MuiTypography-body2 css-1g1qlhj-MuiTypography-root"
            >
              LEADER INDEX
            </p>
          </div>
          <div />
        </div>
      </div>
    </div>
  </div>,
  <div
    className="MuiBox-root css-9gcqgu"
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-spacing-md-2 MuiGrid-spacing-lg-2 css-1ffx5kp-MuiGrid-root"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 MuiGrid-grid-sm-4 MuiGrid-grid-md-4 MuiGrid-grid-lg-7.3 css-juqo8-MuiGrid-root"
      >
        <div
          className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-1xsv8fd-MuiPaper-root"
        >
          <div
            className="MuiGrid-root css-vj1n65-MuiGrid-root"
          >
            <p
              className="MuiTypography-root MuiTypography-body1 css-147w4gn-MuiTypography-root"
            >
              NIFTY 50 
            </p>
          </div>
          <div>
            <div
              className="MuiGrid-root css-1hp6q6g-MuiGrid-root"
            >
              <button
                className="MuiButton-root MuiButton-white MuiButton-whitePrimary MuiButton-sizeMedium MuiButton-whiteSizeMedium MuiButtonBase-root  css-a3xx3e-MuiButtonBase-root-MuiButton-root"
                disabled={false}
                onBlur={[Function]}
                onClick={[Function]}
                onContextMenu={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "background": "white",
                    "color": "black",
                    "margin": "5px",
                    "marginBottom": "0",
                  }
                }
                tabIndex={0}
                type="button"
              >
                3D
                <span
                  className="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
              <button
                className="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root  css-1vbvugf-MuiButtonBase-root-MuiButton-root"
                disabled={false}
                onBlur={[Function]}
                onClick={[Function]}
                onContextMenu={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "background": "white",
                    "color": "black",
                    "margin": "5px",
                    "marginBottom": "0",
                  }
                }
                tabIndex={0}
                type="button"
              >
                1W
                <span
                  className="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
              <button
                className="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root  css-jcwuwq-MuiButtonBase-root-MuiButton-root"
                disabled={false}
                onBlur={[Function]}
                onClick={[Function]}
                onContextMenu={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "background": "white",
                    "color": "black",
                    "margin": "5px",
                    "marginBottom": "0",
                  }
                }
                tabIndex={0}
                type="button"
              >
                1M
                <span
                  className="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
              <canvas
                height={150}
                role="img"
                width={300}
              />
                    
            </div>
          </div>
           
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 MuiGrid-grid-sm-4 MuiGrid-grid-md-4 MuiGrid-grid-lg-7.3 css-5slktz-MuiGrid-root"
      >
        <div
          alignItems="center"
          className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-1xsv8fd-MuiPaper-root"
        >
          <div
            className="MuiGrid-root css-vj1n65-MuiGrid-root"
          >
            <p
              className="MuiTypography-root MuiTypography-body1 css-147w4gn-MuiTypography-root"
            >
              BSE SENSEX 
            </p>
          </div>
          <div>
            <div
              className="MuiGrid-root css-1hp6q6g-MuiGrid-root"
            >
              <button
                className="MuiButton-root MuiButton-white MuiButton-whitePrimary MuiButton-sizeMedium MuiButton-whiteSizeMedium MuiButtonBase-root  css-a3xx3e-MuiButtonBase-root-MuiButton-root"
                disabled={false}
                onBlur={[Function]}
                onClick={[Function]}
                onContextMenu={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "background": "white",
                    "color": "black",
                    "margin": "5px",
                    "marginBottom": "0",
                  }
                }
                tabIndex={0}
                type="button"
              >
                3D
                <span
                  className="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
              <button
                className="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root  css-1vbvugf-MuiButtonBase-root-MuiButton-root"
                disabled={false}
                onBlur={[Function]}
                onClick={[Function]}
                onContextMenu={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "background": "white",
                    "color": "black",
                    "margin": "5px",
                    "marginBottom": "0",
                  }
                }
                tabIndex={0}
                type="button"
              >
                1W
                <span
                  className="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
              <button
                className="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root  css-jcwuwq-MuiButtonBase-root-MuiButton-root"
                disabled={false}
                onBlur={[Function]}
                onClick={[Function]}
                onContextMenu={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "background": "white",
                    "color": "black",
                    "margin": "5px",
                    "marginBottom": "0",
                  }
                }
                tabIndex={0}
                type="button"
              >
                1M
                <span
                  className="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
              <canvas
                height={150}
                role="img"
                width={300}
              />
                    
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="MuiBox-root css-6hex1y"
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-spacing-md-2 MuiGrid-spacing-lg-2 css-1ffx5kp-MuiGrid-root"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 MuiGrid-grid-sm-4 MuiGrid-grid-md-6 MuiGrid-grid-lg-7.3 css-1qxrily-MuiGrid-root"
      >
        <div
          className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-1xsv8fd-MuiPaper-root"
        >
          <div
            className="MuiGrid-root css-vj1n65-MuiGrid-root"
          >
            <p
              className="MuiTypography-root MuiTypography-body1 css-147w4gn-MuiTypography-root"
            >
              TOP GAINERS 
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="ArrowUpwardIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="m4 12 1.41 1.41L11 7.83V20h2V7.83l5.58 5.59L20 12l-8-8-8 8z"
                />
              </svg>
            </p>
          </div>
          <div>
            <div
              className="MuiBox-root css-12pbg1u"
            >
              <div
                className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-spacing-md-1 MuiGrid-spacing-lg-2 css-7gy31f-MuiGrid-root"
              >
                <div
                  className="MuiTableContainer-root css-rorn0c-MuiTableContainer-root"
                  style={
                    Object {
                      "height": 400,
                      "width": 600,
                    }
                  }
                >
                  <table
                    className="MuiTable-root css-f8j0s3-MuiTable-root"
                    role={null}
                    style={
                      Object {
                        "border": "1px solid #000",
                        "borderBottom": "1px solid #000",
                      }
                    }
                  >
                    <thead
                      className="MuiTableHead-root css-5uv86a-MuiTableHead-root"
                      role={null}
                    >
                      <tr
                        className="MuiTableRow-root MuiTableRow-head css-1pgphf0-MuiTableRow-root"
                        role={null}
                      >
                        <th
                          aria-sort={null}
                          className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-14r9yv3-MuiTableCell-root"
                          scope="col"
                        >
                           Name 
                        </th>
                        <th
                          aria-sort={null}
                          className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-14r9yv3-MuiTableCell-root"
                          scope="col"
                        >
                           Change 
                        </th>
                        <th
                          aria-sort={null}
                          className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-14r9yv3-MuiTableCell-root"
                          scope="col"
                        >
                           Sector 
                        </th>
                      </tr>
                    </thead>
                    <tbody
                      className="MuiTableBody-root css-apqrd9-MuiTableBody-root"
                      role={null}
                    />
                  </table>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 MuiGrid-grid-sm-4 MuiGrid-grid-md-6 MuiGrid-grid-lg-7.3 css-1qxrily-MuiGrid-root"
      >
        <div
          className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-1xsv8fd-MuiPaper-root"
        >
          <div
            className="MuiGrid-root css-vj1n65-MuiGrid-root"
          >
            <p
              className="MuiTypography-root MuiTypography-body1 css-147w4gn-MuiTypography-root"
            >
              TOP LOSERS 
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="ArrowDownwardIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="m20 12-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                />
              </svg>
            </p>
          </div>
          <div>
            <div
              className="MuiBox-root css-12pbg1u"
            >
              <div
                className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-spacing-md-1 MuiGrid-spacing-lg-2 css-7gy31f-MuiGrid-root"
              >
                <div
                  className="MuiTableContainer-root css-rorn0c-MuiTableContainer-root"
                  style={
                    Object {
                      "height": 400,
                      "width": 600,
                    }
                  }
                >
                  <table
                    className="MuiTable-root css-f8j0s3-MuiTable-root"
                    role={null}
                    style={
                      Object {
                        "border": "1px solid #000",
                        "borderBottom": "1px solid #000",
                      }
                    }
                  >
                    <thead
                      className="MuiTableHead-root css-5uv86a-MuiTableHead-root"
                      role={null}
                    >
                      <tr
                        className="MuiTableRow-root MuiTableRow-head css-1pgphf0-MuiTableRow-root"
                        role={null}
                      >
                        <th
                          aria-sort={null}
                          className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-14r9yv3-MuiTableCell-root"
                          scope="col"
                        >
                           Name 
                        </th>
                        <th
                          aria-sort={null}
                          className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-14r9yv3-MuiTableCell-root"
                          scope="col"
                        >
                           Change 
                        </th>
                        <th
                          aria-sort={null}
                          className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-14r9yv3-MuiTableCell-root"
                          scope="col"
                        >
                           Sector 
                        </th>
                      </tr>
                    </thead>
                    <tbody
                      className="MuiTableBody-root css-apqrd9-MuiTableBody-root"
                      role={null}
                    />
                  </table>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
]
`;
